lib/stat_tracker.rb

```diff
@@ -37,8 +37,37 @@ def percentage_ties
     @games.values_at(:away_goals, :home_goals).each do |game|
       ties += 1 if game[0] == game[1]
     end
-
     ((ties/total_games)*100).round(1)
   end
 
+  def percentage_home_wins
+    percentage = (home_wins/home_games) * 100
+    p percentage
+  end
+
+  def game_wins
+    win = []
+    @game_teams[:result].map {|row| win << row   if row == "WIN"}; p win.count.to_f
+  end 
+
```

Comment:
I'd like to see a know a little more about these helper methods:

- What is the semi-colon doing before the p? Also it does not look like we need the "p's" since they would be the return value and we don't need to print them. 

- Also have you thought about doing `win = 0.0 ` vs `win = []` this way we are not storing an array of multiple strings of "WIN" we could just incramint that count up one as you iterate through?  

- Then there would not be a need to use the `.count` or `.to_f` method you would just call the `win` variable at the end. 

- Final thought should the variable win be plural since it is tracking multiple wins not just one win?

Reply:
Could we `select` the games where the `:result` is `WIN`?

Reply:
ok made these updates! thanks!